From 3c219718ebda20b9de9e1cb01f5b94138a7555b1 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Tue, 7 Apr 2020 15:15:42 +0800
Subject: [PATCH 3/3] s32g274a: dtsi: correct opad regs size

Based on the user manual, the corresponding registers of opad1
and opad2 size are 0xc. Here correct it to avoid accessing them
from userspace will trigger external abort.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-s32g274a.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-s32g274a.dtsi b/arch/arm64/boot/dts/freescale/fsl-s32g274a.dtsi
index 53d58dc2e4f8..f7b264d3ca9e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-s32g274a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-s32g274a.dtsi
@@ -356,12 +356,12 @@
 	};
 	opad1_regs: siul2_reg@0x4401170C {
 		compatible = "fsl,opad0_reqs-s32gen1","syscon","simple-mfd";
-		reg = <0x0 0x4401170C 0x0 0x14>;
+		reg = <0x0 0x4401170C 0x0 0xc>;
 		big-endian;
 	};
 	ipad1_regs: siul2_reg@0x4401174C {
 		compatible = "fsl,ipad0_reqs-s32gen1","syscon","simple-mfd";
-		reg = <0x0 0x4401174C 0x0 0x14>;
+		reg = <0x0 0x4401174C 0x0 0xc>;
 		big-endian;
 	};
 
-- 
2.17.1

